src/main/java/in/muthukumari/controller/BankDetailsController.java

```diff
@@ -1,35 +1,26 @@
 package in.muthukumari.controller;
 
-import java.sql.SQLException;
-import java.util.Map;
 import java.util.Set;
 import java.util.logging.Logger;
-
-import in.muthukumari.dao.BankDAO;
-import in.muthukumari.exception.DBException;
-import in.muthukumari.exception.InvalidException;
+import in.muthukumari.exception.ServiceException;
 import in.muthukumari.service.BankDetailService;
 
 public class BankDetailsController {
 	
 	final Logger logger =  Logger.getLogger(this.getClass().getName());
 	
-	public BankDetailsController() {
-		//Default constructor
-	}
-	
 	/**
 	 * This method used to get the branch name list of each bank
 	 * @param bankName
 	 * @return
-	 * @throws ClassNotFoundException
-	 * @throws SQLException
 	 */
-	public static Set<String> getBranchNameList(String bankName) throws ClassNotFoundException, SQLException {
```

Comment:
handle ClassNotFoundException, SQLException  in dao itself

Reply:
Already I did changed this.